tests/jballerina-unit-test/src/test/resources/test-src/types/byte/byte-array-value-negative.bal

```diff
@@ -17,3 +17,17 @@ function base64InvalidLiteralTest() {
     byte[] f = base64 `afcd341a4bdfaaabmcfccadabcd89 12df ss==`;
     byte[] g = base64 `afcd34abcdef123aGc2?>><*&*^&34bcd1a4bdfABbadaBCd892s3as==`;
 }
+
+function byteArrayLiteralTypeTest() {
+    byte[2] a = base16 `aa bb`;
+    byte[3] b = base16 `aa bb`;             // error
+    int[3] c = base16 `aa bb`;              // error
+    byte[*] d = base16 `aa bb`;
```

Comment:
Do we need this lines for the negative tests, because of there is no error

Reply:
Can't we have positive cases in negative test? @MaryamZi @pcnfernando